src/WCSimEnumerations.cc

```diff
@@ -100,6 +101,78 @@ std::string WCSimEnumerations::EnumAsString(ReflectionSurface_t r)
   }
   return "";
 }
+ 
+std::string WCSimEnumerations::EnumAsString(ProcessType_t p)
+{
+  switch(p) {
+  case (kHadElastic) :
+    return "hadElastic";
+    break;
+  case (kNCapture) :
+    return "nCapture";
+    break;
+  case (kCompt) :
+    return "compt";
+    break;
+  case (kPhot) :
+    return "phot";
+    break;
+  case (kCCerenkov) :
+    return "Cerenkov";
+    break;
+  case (kSScintillation) :
+    return "Scintillation";
+    break;
+  case (kEBrem) :
+    return "eBrem";
+    break;
+  case (kAnnihil) :
+    return "annihil";
+    break;
+  case (kConv) :
+    return "conv";
+    break;
+  case (kEIoni) :
+    return "eIoni";
+    break;
+  case (kNeutronInelastic) :
+    return "neutronInelastic";
+    break;
+  default: 
+    return "";
+    break;
+  }
+  return "";
+}
+
+ProcessType_t WCSimEnumerations::ProcessTypeStringToEnum(const std::string& str) {
+  if (str == "hadElastic") {
+    return kHadElastic;
+  } else if (str == "nCapture") {
+    return kNCapture;
+  } else if (str == "compt") {
+    return kCompt;
+  } else if (str == "phot") {
+    return kPhot;
+  } else if (str == "Cerenkov") {
```

Comment:
```suggestion
  } else if (str == "Cherenkov") {
```
This is just the way we spell it in HK